packages/example-app-base/src/components/templates/PageImageSet.tsx

```diff
@@ -41,10 +41,9 @@ export class PageImageSet extends BaseComponent<IPageImageSetProps, {}> {
   public render(): JSX.Element {
     const { theme, getStyles, images } = this.props;
     const styleProps: IPageImageSetStyleProps = { theme: theme! };
-    const classNames: IClassNames<IPageImageSetStyles> = mergeStyleSets(
-      getDefaultStyles(styleProps),
-      getStyles && getStyles(styleProps)
-    );
+    const classNames = getStyles
```

Comment:
This LOOKS cleaner - but is there something fundamentally wrong with having a null as a 2nd param?

Reply:
There is a type error.. I didn't change this because I was too free 😁 

`example-app-base` could use a lot of work to get spruced up to modern standards. In modern standards, we don't call` mergeStyleSets` from components but we call `classNameFunction`/`getClassNames` instead. dzearing asked me to do the bare minimum though and to invest my time in Dropdown conversion instead.